apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetVersionMapper.java

```diff
@@ -20,6 +21,9 @@ interface DatasetVersionMapper {
     @Mapping(target = "lastUpdatedBy", source = "userName")
     @Mapping(target = "isLatest", ignore = true)
     @Mapping(target = "tags", ignore = true)
+    @Mapping(target = "versionName", ignore = true)
+    @Mapping(target = "createdAt", ignore = true)
```

Comment:
The createdAt & lastUpdatedAt, are they needed mappers?

PR is on versionName

Makes sense if all fields should be mapped

Reply:
As this is a summary, we don't need all properties. These are ignored by intention.